build.sbt

```diff
@@ -12,8 +12,8 @@ lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
     // Semicolon-separated list of regexs matching classes to exclude
-    ScoverageKeys.coverageExcludedPackages := "<empty>;.*views.*;.*prod.*;.*test.*",
-    ScoverageKeys.coverageExcludedFiles := "<empty>;.*BuildInfo.*;.*Routes.*;",
+    ScoverageKeys.coverageExcludedPackages := "<empty>;.*views.*;.*prod.*;.*test.*;app.routes;testOnlyDoNotUseInAppConf;prod.*;uk.gov.hmrc.digitalservicestax.controllers;uk.gov.hmrc.digitalservicestax.config;",
```

Comment:
won't we need tests for the controllers and the config?

Reply:
@hmrclt That's an outdated copy paste from the other repo. It's already been addressed in my next PR, so we can merge this.

Reply:
@hmrclt Ignore in this PR btw, all fixed in the next one.